### PR TITLE
Improve the script to extract logs

### PR DESCRIPTION
**Pull request by rakanalh**
Mostly, download the logs from 2 servers and not 1.
Also separated the logs into their own date-prefixed directory
so that the base directory doesn't get polluted

## Description

Fixes: #<issue>

Please, describe what this PR does in detail:
- If it's a bug fix, detail the root cause of the bug and how this PR fixes it.
- If it's a new feature, describe the feature this PR is introducing and design decisions.
- If it's a refactoring, describe why it is necessary. What are its pros and cons in respect to the previous code, and other possible design choices.

## PR review check list

Quality check list that cannot be automatically verified.

- Safety
    - [ ] The changes respect the necessary conditions for safety (https://raiden-network-specification.readthedocs.io/en/latest/smart_contracts.html#protocol-values-constraints)
-  Code quality
    - [ ] Error conditions are handled
    - [ ] Exceptions are propagated to the correct parent greenlet
    - [ ] Exceptions are correctly classified as recoverable or unrecoverable
- Compatibility
    - [ ] State changes are forward compatible
    - [ ] Transport messages are backwards and forward compatible
- Commits
    - [ ] Have good messages
    - [ ] Squashed unecessary commits
- If it's a bug fix:
    - Regression test for the bug
        - [ ] Properly covers the bug
        - [ ] If an integration test is used, it could not be written as a unit test
- Documentation
    - [ ] A new CLI flag was introduced, is there documentation that explains usage?
- Specs
    - [ ] If this is a protocol change, are the specs updated accordingly? If so, please link PR from the specs repo.
- Is it a user facing feature/bug fix?
    - [ ] Changelog entry has been added

### CODE REVIEW

File: tools/debugging/extract_sp_exceptions.sh [hunk 1]
```diff
@@ -36,6 +37,9 @@ else
     DESTINATION_DIR=$(pwd)
 fi
 
+DESTINATION_DIR="$(realpath -s ${DESTINATION_DIR})/$(date +%m-%d-%Y)"
+mkdir -p $DESTINATION_DIR
+
 print_bold "Downloading PFS logs"
 
 ssh root@services-dev.raiden.network 'cd raiden-services/deployment/; docker-compose logs pfs-goerli | gzip' > ${DESTINATION_DIR}/pfs.log.gz
```
Comment (palango) [inline tools/debugging/extract_sp_exceptions.sh]: Can you download `pfs-goerli-with-fee` here as well

Comment (rakanalh) [reply tools/debugging/extract_sp_exceptions.sh]: So download both?

Comment (palango) [reply tools/debugging/extract_sp_exceptions.sh]: I'd say so, I think we use both in the scenarios

Comment (rakanalh) [reply tools/debugging/extract_sp_exceptions.sh]: done

**Pull request merged**